rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/360_combined_fields.yml

```diff
@@ -0,0 +1,45 @@
+setup:
+  - do:
+      indices.create:
+        index:  test
+        body:
+          mappings:
+            properties:
+              title:
+                type: text
+              abstract:
+                type: text
+              body:
+                type: text
+
+  - do:
+      index:
+        index: test
+        id: 1
+        body:
+          title: "Time, Clocks and the Ordering of Events in a Distributed System"
+          abstract: "The concept of one event happening before another..."
+          body: "The concept of time is fundamental to our way of thinking..."
+
+  - do:
+      indices.refresh: {}
+
+---
+"Test combined_fields query":
+  - skip:
+      version: " - 7.99.99"
+      reason: "combined fields query is not yet backported"
+  - do:
+      search:
+        rest_total_hits_as_int: true
```

Comment:
As we want to deprecate `rest_total_hits_as_int` parameter, may be we don't need to introduce it in new tests.

Reply:
Oops, this is a good point!